src/main/java/reactor/netty/http/client/HttpClientOperations.java

```diff
@@ -296,7 +305,15 @@ public HttpClientRequest headers(HttpHeaders headers) {
 
 	@Override
 	public boolean isFollowRedirect() {
-		return redirectable && redirectedFrom.length <= MAX_REDIRECTS;
+		return (redirectable ||
+				(followRedirectPredicate != null && followRedirectPredicate.test(this, this))) &&
```

Comment:
Shouldn't invoke on accessor like isFollowRedirect

Reply:
New update is provided. Please take a look.